pkgs/applications/office/fava/default.nix

```diff
@@ -13,6 +13,7 @@ buildPythonApplication rec {
   };
 
   checkInputs = [ python3.pkgs.pytest ];
```

Comment:
```suggestion
  checkInputs = [ python3.pkgs.pytestCheckHook ];
```

Reply:
Not converting that right now.

Reply:
https://github.com/NixOS/nixpkgs/pull/118355